### DISCUSSION TITLE
User interface lacks support for multi-language localization

### DISCUSSION THREAD

**Discussion by ChidiChuks** [btcpayserver/btcpayserver#5801] [Technical Support]
The current user interface lacks support for multi-language localization, making it difficult for non-English-speaking users to navigate and use the platform effectively. The absence of multi-language support in the user interface limits the accessibility of BTC Pay Server to non-English-speaking users, hindering its adoption and usability in diverse global communities.

**Comment by pavlenex** [comment]
We're aware of this already, and it's tagged for 1.14.0 release. Would you like to help us test it? https://github.com/btcpayserver/btcpayserver/pull/5662
